src/color.rs

```diff
@@ -1,5 +1,140 @@
+use std::{fmt, str::FromStr};
+
 use tui::style::Color;
 
+pub enum AcceptedColors {
```

Comment:
Just out of curiosity: Why not using a function like the following instead of defining a new enum?
```
fn next_color(color: Color) -> Color {
    match color {
        Color::Black => Color::Red,
        ...
}
```

Reply:
I believe either way is fine, with an enum I just kept all the related functions tied together.